theme/management/commands/migrate_quota_holder_from_irods.py

```diff
@@ -0,0 +1,59 @@
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+from django_irods.storage import IrodsStorage, SessionException
+from hs_core.models import BaseResource
+
+
+def get_quota_holder(res):
+    """Get quota holder of the resource.
+    return User instance of the quota holder for the resource or None if it does not exist
+    """
+    try:
+        uname = res.getAVU("quotaUserName")
+    except SessionException:
+        # quotaUserName AVU does not exist, return None
+        return None
+
+    if uname:
+        return User.objects.filter(username=uname).first()
+    else:
+        # quotaUserName AVU does not exist, return None
+        return None
+
+
+class Command(BaseCommand):
+    help = "Migrate quota_holder info from irods AVU into Django UserQuota model."
```

Comment:
command to migrate the quota_holder metadata from irods AVU into django resource model